modules/core/src/adapter/resources/shader.ts

```diff
@@ -117,7 +117,10 @@ ${htmlLog}
     button.style.textAlign = 'left';
     document.body.appendChild(button);
 
-    document.getElementsByClassName('luma-compiler-log-error')[0]?.scrollIntoView();
+    const errors = document.getElementsByClassName('luma-compiler-log-error');
+    if (errors[0]?.scrollIntoView) {
+      errors[0].scrollIntoView();
```

Comment:
Just curious,  I think the following should work
```suggestion
      errors[0]?.scrollIntoView?.();
```

Reply:
Yes it does, didn't know about that notation. But I just can't convince myself that it is more readable